test_utils/factories.py

```diff
@@ -44,9 +44,10 @@ class Meta:
         """
 
         model = CourseSkills
-        django_get_or_create = ('course_id', 'skill')
+        django_get_or_create = ('course_key', 'skill')
 
     course_id = factory.LazyAttribute(lambda x: FAKER.slug())
```

Comment:
should we remove this course_id field from this factory model now?

Reply:
Same as above, we can not remove as doing that will case integrity error on tests.